package.json

```diff
@@ -51,6 +51,7 @@
     "tsc": "tsc --noEmit",
     "postinstall": "patch-package",
     "coverage-if-changed": "./scripts/run-coverage-if-changed.sh npm run coverage",
+    "postpack": "./scripts/postpack",
     "prepare": "./scripts/prepublish"
```

Comment:
do we need this "prepare" for anything other than pack? if not, it can be renamed to "prepack", and then it runs only before pack..